go/common/common.go

```diff
@@ -5,7 +5,7 @@ import (
 	"github.com/google/uuid"
 )
 
-type NodeID uint64
+type NodeID common.Address
```

Comment:
I personally don't like this alias. I think it obscures from than it clarifies.

Reply:
yes. I'll get rid of it